ostree/gen-fixtures.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+DIR=""
+TREE=$(mktemp -d /tmp/tree.XXXXX)
+USAGE="
+Usage: gen-fixtures.sh <output-dir>
+
+Generate an OSTree repository with (2) branches:
+ - rawhide
+ - stable
+
+Each branch will have (3) commits for versions:
+ - 1.0
+ - 1.1
+ - 1.2
+
+The OSTree repository will be created in <output-dir>.
+The directory will be be created.
+"
+
+show_help()
+{
+  echo "${USAGE}"
+}
+
+_mkdir()
+{
+  if [ -e ${DIR} ]; then
+    if [ ! -z "$(ls -A ${DIR})" ]; then
+      echo 1>&2 ""
+      echo 1>&2 "${DIR} must be empty."
+      echo 1>&2 ""
+      exit 1
+    fi
+  else
+    mkdir -p "${DIR}"
+  fi
+}
+
+gen_tree()
```

Comment:
I'm wondering what the purpose of this `gen_tree()` function is. I know that it (hopefully) copies this script into `/tmp/tree.XXXXX`, but I don't understand what purpose it serves. I can comment out the call to this function, and the end result seems the same.